ambari-infra/ambari-infra-solr-client/src/main/java/org/apache/ambari/infra/solr/AmbariSolrCloudCLI.java

```diff
@@ -62,6 +63,7 @@
       + "\n./solrCloudCli.sh --remove-admin-handlers -z host1:2181,host2:2181/ambari-solr -c collection"
       + "\n./solrCloudCli.sh --create-znode -z host1:2181,host2:2181 -zn /ambari-solr"
       + "\n./solrCloudCli.sh --check-znode -z host1:2181,host2:2181 -zn /ambari-solr"
+      + "\n./solrCloudCli.sh --transfer-znode -z host1:2181,host2:2181 -tz -cps /ambari-solr -cpd /ambari-solr-backup"
```

Comment:
Can you please clarify this example?  I think `-tz` is duplicated (as it's the short form of `--transfer-znode`), but `-tm` does not appear anywhere, nor does its argument.